lib/scope.js

```diff
@@ -402,17 +438,29 @@ module.exports = {
        let noneOfTheAboveCheckbox = fieldset.querySelector('input.danota-checkbox');
```

Comment:
Make clear somehow that only one of `base64_id_of_first_choice` and `base64_id_of_index` will exist. The way we check for both made it seem to me like both of these might be true somehow. Same for `base64_var_name` and `base64_index`.

Reply:
This would be negated by using the second comment to clarify the code. Thanks!